packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/html/tags.element.scope

```diff
@@ -0,0 +1,20 @@
+<div>hello</div>
+---
+
+[#1.1 Content] =
+[#1.1 Removal] = 0:0-0:5
+0| <div>hello</div>
+  >-----<
+
+[#1.1 Insertion delimiter] = " "
+
+[#1.2 Content] =
+[#1.2 Removal] = 0:10-0:16
+0| <div>hello</div>
+            >------<
+
+[#1.2 Insertion delimiter] = " "
+
+[Domain] = 0:0-0:16
```

Comment:
Hmm it's a bit confusing that the above have a `#1` but this one doesn't. I wonder if we should do `.1` instead of `#1.1` if there's only one?

Reply:
We can do that. `.1` or `#.1`?

Reply:
done